rosdep/base.yaml

```
@@ -4094,6 +4094,12 @@ libnanoflann-dev:
     '*': [libnanoflann-dev]
     bionic: null
     focal: null
+libnanopb-dev:
+  debian:
+    bullseye: [libnanopb-dev]
```

Since this is available in all stable versions of Debian, we can just use:

```suggestion
  debian: [libnanopb-dev]
```